octopus_deploy/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": "2.0.0",
   "app_uuid": "821889b0-d4f9-4136-8059-7b7c42e6bd43",
   "app_id": "octopus-deploy",
-  "display_on_public_website": false,
```

Comment:
Shouldn't this be handled now through SDP feature flags? I see in SDP that this one does not have any outside of staging.

Reply:
SDP controls the tile, this handles whether or not the docs are public on our website ie https://docs.datadoghq.com/integrations/solr/?tab=host

Reply:
Oh ok, I think I was confused with the internal `OVERRIDE_APP_MANIFEST` on the legacy Config. That one seems to basically control the tile mostly.